tools/ports/libpng.py

```diff
@@ -19,7 +18,7 @@ def needed(settings):
 
 
 def get(ports, settings, shared):
-  ports.fetch_project('libpng', 'https://github.com/emscripten-ports/libpng/archive/' + TAG + '.zip', 'libpng-' + TAG, sha512hash=HASH)
+  ports.fetch_project('libpng', 'https://download.sourceforge.net/libpng/libpng-' + TAG + '.tar.gz', 'libpng-' + TAG, sha512hash=HASH)
```

Comment:
We have had problems with sourceforge downloads in the past.   I'd rather not rely on their uptime.

The solution we normally have is to mirror the archive on our own google storage bucket.

See #13935 and #13878.

If you would like me upload a certain version of libpng to this bucket just let me know and I can do that.

Reply:
Would we be better off updating the emscripten-port's copy, then? It's 6 vs half-dozen.

If you want to store it in a Google bucket, this is the most recent file from the libpng.org site:
https://download.sourceforge.net/libpng/libpng-1.6.37.tar.gz

Reply:
I uploaded to https://storage.googleapis.com/webassembly/emscripten-ports/libpng-1.6.37.tar.gz.

The SHASUM should be identical.  

I think its nicer to use a mirror of the pristine sources when we have no patches to maintain.    Have a separate github repo makes it look like we have/need patches.

Reply:
I updated the script to use that, but it's still failing in CircleCI.